### DISCUSSION TITLE
Home Assistant - How to connect to RaspberryMatic GUI from out of HA?

### DISCUSSION THREAD

**Discussion by acidoverflow** [OpenCCU/OpenCCU#1447] [🇺🇸 Q&A] [answered]
Hi, today I successfully managed to merge our standalone PI3 RasberryMatic installation into HA. Everything seems to work fine. 
There is only one problem when I try to open the RasberryMatic GUI directly without using HA. How can I manage to do that? I think I have to connect to the host IP Adress and then somehow have to forward my request. Where can I set that up? In the configuration tab where there is the container host and description? What host should I fill in there? It doesn't take the internal hostname nor can I add an IP as it is always cut off.
  
Thanks for help in advance. 

Arne

**Comment by acidoverflow** [comment, marked as answer]
Problem solved. I simple had to fill in the ports I wanted to route to the RaspberryMatic Instance.

**Comment by marc965** [comment]
Hi,
please tell me how you do that!
I am only accessing Raspberry Matic GUI over HA.
Thanks for help.

**Reply by acidoverflow** [reply]
Hi Marc, as you can see I just put in the port 443 in the configurations tab of the integration. After a reboot I‘m able to connect directly to the GUI without having to use HA. HTTPS://IPofHA:443 . I also use it for the @home App. ![1F2C5414-A726-4AC9-92AB-0221AC769106](https://user-images.githubusercontent.com/154293/138339922-1f28cb1c-da85-41bc-ba45-f493b678accd.png)

**Comment by marc965** [comment]
Danke.
Funktioniert.
Allerdings habe ich nicht verstanden was da genau passiert, geschweige denn verstanden für was die anderen Ports genutzt werden können.

Thank you.
It works now, but i dont understand how this works internally...

Marc.

**Reply by acidoverflow** [reply]
I think this may help https://developers.home-assistant.io/docs/add-ons/presentation/#ingress

**Comment by marc965** [comment]
Thank you for your answer.
This helps me to understand how works HA internally....
